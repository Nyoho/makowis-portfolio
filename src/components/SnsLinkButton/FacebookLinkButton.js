/* eslint-disable react/prop-types */
// @flow
import React from 'react';
import BaseButton from './BaseButton';

const FacebookLinkButton = (props: { url: string }) => {
  const btnstyle = {
    background: '#3b75d4',
  };
  return <BaseButton url={props.url} icon="facebook-f" btnstyle={btnstyle} />;
};

export default FacebookLinkButton;
