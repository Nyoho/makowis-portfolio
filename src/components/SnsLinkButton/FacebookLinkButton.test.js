/* eslint-env jest */
// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import FacebookLinkButton from './FacebookLinkButton';

it('renders FacebookLinkButton', () => {
  const tree = renderer
    .create(<FacebookLinkButton url="http://www.facebook.com/makoto.henmi" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
