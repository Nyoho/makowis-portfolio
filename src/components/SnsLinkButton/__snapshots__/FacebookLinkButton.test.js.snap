// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders FacebookLinkButton 1`] = `
<a
  className="SnsLinkButton"
  href="http://www.facebook.com/makoto.henmi"
  style={
    Object {
      "background": "#3b75d4",
    }
  }
/>
`;
